src/components/Footer.jsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Button, Flex } from "@chakra-ui/react";
+import React from "react";
+
+export default function Footer() {
+  return (
+    <div>
+      <Box marginTop={10}>
+        <Flex gap={4} justifyContent="center" width="80%" margin="auto">
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            Meta
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            About
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            Blog
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            Jobs
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            Help
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            API
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            Privacy
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            Terms
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            Top Accounts
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            Hashtags
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            Locations
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
```

Comment:
You have many buttons of same style. You can include them in array and map over them. Would help in reducing repetition. Just a suggestion

Reply:
Good point, will update it soon
Thanks!